packages/web-react/src/components/Alert/Alert.tsx

```diff
@@ -17,6 +17,15 @@ export const Alert = <T extends ElementType = 'div', E = void>(props: SpiritAler
   const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
   const alertIconName = useAlertIcon({ color, iconName, ...otherProps });
 
+  useDeprecationMessage({
+    method: 'custom',
+    trigger: color === 'danger',
```

Comment:
can we somehow check if the assets already contain the danger icon? And if so, we don't show this message?

Reply:
I see we can do this in the useIcon, can we implement this here too?

Reply:
Icon has access to the IconContext. Alert not. We can use `useIcon` hook to get the information but then we will be calling the icon twice.

Reply:
:( But then we can hide the deprecation message to the user who already has the icon... :)

I still don't like the unavoidable deprecation messages (especially when we are adding quite a few of them), but I will leave it up to you.

Reply:
Ok, you are right about the messages and that we can avoid them even at the cost of a few more lines of code written. The code will be refactored when the next major version is released.